api-url.js

```diff
@@ -0,0 +1,2 @@
+import { ApiUrl } from './src/ApiUrl.js';
```

Comment:
Please, add HTML tag typescript definition (see authorization method component for an example)